.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+sudo: required
+dist: trusty
+before_script:
+  - npm run compile
+before_install:
+  - docker pull selenium/standalone-chrome
```

Comment:
do we need the docker stuff for terra-site tests?

Reply:
Atm no.